sdk/storage/azure-storage-file-share/src/main/java/com/azure/storage/file/share/models/ShareProtocolSettings.java

```diff
@@ -11,7 +11,7 @@
 /**
  * Protocol settings.
  */
-@JacksonXmlRootElement(localName = "ShareProtocolSettings")
+@JacksonXmlRootElement(localName = "ProtocolSettings")
```

Comment:
Was this a bug in Swagger/codegen or a change on the service side to change the XML element name?

Reply:
bug in swagger